app/modules/[moduleName]/loading.tsx

```
@@ -14,11 +14,11 @@ export default function ModuleDetailsLoading() {
       <div className="flex items-center justify-between py-2">
         <div className="flex items-center">
           {/* Title skeleton */}
-          <div className="h-8 w-48 bg-muted rounded ml-2 mb-2"></div>
+          <div className="h-8 w-48 addmarginforheaders bg-muted rounded ml-2 mb-2"></div>
```

Biome (1.9.4)

[error] JSX elements without children should be marked as self-closing. In JSX, it is valid for any element to be self-closing.

Unsafe fix: Use a SelfClosingElement instead


(lint/style/useSelfClosingElements)